CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - [SIL.Chorus.LibChorus] Add webm as additional audio file type
+- [SIL.Chorus] Add ability to clone project without direct user interaction
```

Comment:
"without user interaction": there's no indirect interaction, either.

Reply:
In my mind, the indirect user interaction is them using the wrapping software. They still have to do something to get the clone to happen.